tests/Doctrine/Tests/Common/Collections/ArrayCollectionTest.php

```diff
@@ -18,4 +23,35 @@ protected function buildCollection(array $elements = []) : Collection
     {
         return new ArrayCollection($elements);
     }
+
+    public function testUnserializeEmptyArrayCollection() : void
+    {
+        $collection            = new SerializableArrayCollection();
+        $serializeCollection   = serialize($collection);
+        $unserializeCollection = unserialize($serializeCollection);
+
+        $this->assertIsArray($unserializeCollection->getValues());
+        $this->assertCount(0, $unserializeCollection->getValues());
+    }
+}
+
+/**
+ * We can't implement Serializable interface on anonymous class
+ */
+class SerializableArrayCollection extends ArrayCollection implements Serializable
+{
+    public function serialize() : string
+    {
+        return json_encode($this->getKeys());
+    }
+
+    /**
+     * @param string $serialized
```

Comment:
Maybe there is a syntax to ignore specific phpcs warnings that could help in there: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Advanced-Usage#ignoring-parts-of-a-file

Reply:
Thanks for the tip